lambda/yara_av.tf

```diff
@@ -61,3 +76,22 @@ resource "aws_iam_role_policy_attachment" "lambda_role_policy" {
   policy_arn = aws_iam_policy.lambda_policy.*.arn[0]
   role       = aws_iam_role.lambda_iam_role.*.name[0]
 }
+
+resource "aws_security_group" "allow_efs_lambda_av" {
+  count       = local.count_av_yara
+  name        = "allow-efs"
```

Comment:
Can you give it a more specific name? I think there's already an `allow-efs` somewhere

Reply:
Fair point